Library/Homebrew/extend/os/linux/dependency_collector.rb

```diff
@@ -0,0 +1,16 @@
+# typed: true
+# frozen_string_literal: true
+
+class DependencyCollector
+  undef parse_symbol_spec
+
+  private
+
+  def parse_symbol_spec(spec, tags)
```

Comment:
I think this should be the default logic rather than Linux-specific and only have a macOS override.